### PR TITLE
[TASK] Ignore more files from uploading

### PR DESCRIPTION
**Pull request by oliverklee**
These files also occur in some extensions (including the Tea example
extension by the TYPO3 documentation team).

### CODE REVIEW

**Comment by bmack** [comment]
Can you rebase on latest main branch?

**Comment by oliverklee** [comment]
Rebased and repushed.

**Review by bmack — APPROVED** [review]

File: conf/ExcludeFromPackaging.php [hunk 1]
```diff
@@ -24,9 +24,11 @@
         'vendor',
     ],
     'files' => [
+        'CODE_OF_CONDUCT.md',
         'DS_Store',
         'Dockerfile',
         'ExtensionBuilder.json',
+        'ExtensionBuilder.json',
```
Comment (o-ba) [inline conf/ExcludeFromPackaging.php:31]: Why adding this a second time?

Comment (oliverklee) [reply conf/ExcludeFromPackaging.php:31]: Oh, this was my mistake when I resolved the merge conflict. I'll fix this in a minute. Good catch, thanks!

Comment (oliverklee) [reply conf/ExcludeFromPackaging.php:31]: #15

**Pull request merged**